splunk_test.go

```diff
@@ -1411,7 +1412,7 @@ func TestCannotSendAfterClose(t *testing.T) {
 		t.Fatal("Driver should not allow to send messages after close")
 	}
 
-	if len(hec.messages) != 1 {
+	if !(len(hec.messages) > 0) {
```

Comment:
why is the logic modified here from length != 1 condition?

Reply:
That was a mistake. Reverted to the original test